### DISCUSSION TITLE
Docset integration with Zeal

### DISCUSSION THREAD

**Discussion by eclectic-coding** [tailwindlabs/tailwindcss#2217] [Ideas]
I have been using a local documentation app called [Zeal](https://zealdocs.org/) which supports the Docset format of documentation via Dash. This app allows me to have documentation on my desktop. I currently have documentations locally for CSS, HTML, Python3, React, Rails, and Vue. 

It would be wonderful to add Tailwind to this list. 

[Docset documentation](https://kapeli.com/docsets)

**Comment by p1613m** [comment]
Please, add

**Comment by csanso-limit** [comment]
I need this actually. Because at the international WorldSkills competition Zeal Docs will be the only way to access documentation. As internet access is forbidden.

**Reply by horanchikk** [reply]
this really helpful for worldskills, +1

**Reply by TravisRoad** [reply]
+1

**Comment by andreslucena** [comment]
I found this project that seems to work well: https://github.com/godbout/tailwindcss-dash-docset

To use it, there are [Releases with the docset folder](https://github.com/godbout/tailwindcss-dash-docset/releases).

**Reply by Adrian-Benavente** [reply]
That's good for Dash, but for Zeal a "feed URL" is required, there's no option to import the .docset folder at least in my version of the software.

**Reply by ghost** [reply]
Any updates on this? How to build a feed?

**Reply by stevenleeg** [reply]
Just got this working! You can download the `tailwindcss.tgz` file from the releases page then copy the `tailwindcss.docset` directory within the archive to Zeal's docset directory (you can find this path from `Edit > Preferences > Docset storage`). Restart Zeal and you should see it in the sidebar.

**Comment by midnight-wonderer** [comment]
### Central Repository
This is available on Dash's user_contrib repository.
The list is somewhat hard to access; you may browse available packages [here](https://kapeli.com/feeds/zzz/user_contributed/build/index.json). [[ref]](https://github.com/zealdocs/zeal/issues/170#issuecomment-71310603)

### Adding to Zeal
1. Note down where Zeal stores DocSets (visible in the settings). [[ref]](https://github.com/zealdocs/zeal/issues/644#issuecomment-257288208)
2. $ npm install -g [zeal-user-contrib](https://github.com/jmerle/zeal-user-contrib)
3. $ zeal-user-contrib --install `Tailwind_CSS` --output-directory `<What you noted in step 1>`
